frontend/packages/integration-tests-cypress/tests/app/node-terminal.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { checkErrors } from '../../support';
+import { detailsPage } from '../../views/details-page';
+import { listPage } from '../../views/list-page';
+
+describe('Node terminal', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Opens a debug terminal', () => {
```

Comment:
If you haven't, it would be good to temporarily comment out your change from NodeTerminal.tsx and make sure this fails.

Reply:
Good suggestion.  Failure confirmed.

![Screenshot 2022-11-29 at 3 23 56 PM](https://user-images.githubusercontent.com/895728/204640079-94f93d78-20d4-4a05-93a0-abd42a50ad42.png)

/hold cancel